docs/_tutorials/bert-finetuning.md

```diff
@@ -7,7 +7,19 @@ In this tutorial we will be adding DeepSpeed to the BingBert model for the SQuAD
 
 ## Overview
 
-Please clone the repository and go to the `examples/BingBertSquad` folder to follow along.
+Please clone the DeepSpeed repository and change to deepspeed directory
+
+`git clone https://github.com/microsoft/deepspeed`
+
+`cd deepspeed`
+
+The DeepSpeedExamples are submodules so you need to initialize and update them using the following commands
+
+`git submodule init`
+
+`git submodule update`
```

Comment:
We could also do this in a single command (this is what we do in our install script also): `git submodule update --init --recursive`.

Reply:
I agree. Both methods are fine so I guess I will keep the ones I have and mark this as resolved.